pb/custom_evaluator/custom_evaluator.proto

```diff
@@ -52,6 +52,10 @@ message TestResultComparison {
     //
     // NOTE: Only supported for string and numerical values
     string test_result_error_field = 4;
+
+    // This will enable the target status comparation. The value of the status
+    // will be used to evaluate.
+    bool target_status_field = 5;
```

Comment:
nit: maybe `target_status` instead since this isn't the name of a field, but the field itself

Reply:
Thanks for the review, will change it in next commit!